airflow/config_templates/config.yml

```diff
@@ -2384,3 +2384,119 @@
       type: string
       example: ~
       default: "NamedHivePartitionSensor"
+- name: mesos
+  description: ~
+  options:
+    - name: master
+      description: /
```

Comment:
For consistency with the rest of the config file please remove the leading `/`

Reply:
Done